### ISSUE TITLE
outdated info on passwordstore.org

### ISSUE THREAD

**Issue by 532910** [browserpass/browserpass-legacy#328]
Yes, I know, this is not a browserpass issue, but I've already wrote to passwordstore mail list without any response:
https://lists.zx2c4.com/pipermail/password-store/2019-February/003593.html

**Comment by maximbaz** [comment]
Hehe thanks for caring about it and trying to update the info on passwordstore.org 🙂 

I wish I could help, but I don't know anyone behind passwordstore.org to ping personally, sad that you don't get a response. You are not the first person who is reporting on that mailing list, see for example [this post](https://lists.zx2c4.com/pipermail/password-store/2017-December/003159.html) from 2017.

I'll close the ticket since it's not actionable for me, but if someone is reading this thread and has an ability to update passwordstore.org, please do so!

**Comment by 532910** [comment]
> but I don't know anyone behind passwordstore.org to ping personally

Jason@zx2c4.com

**Comment by maximbaz** [comment]
He is for sure reading the mailing list, so he has already seen all these reports though 😕

**Comment by 532910** [comment]
Yes, but just may be I'm in his spamlist or some other blocklist.

**Comment by maximbaz** [comment]
I'm pretty sure that's not the case (😉), but to support your good cause, once I release v3 I'll reply on your mail saying that the link has changed again and it's now https://github.com/browserpass/browserpass-extension, will CC Jason directly.

For my future reference: `In-Reply-To: <3d339674-4e96-1e06-5904-39dd64f54b05@outerface.net>`

Reopening for the reminder.

**Comment by maximbaz** [comment]
https://lists.zx2c4.com/pipermail/password-store/2019-April/003637.html

**Comment by erayd** [comment]
Turns out some of the emails may have been going into the spam bucket. Re-opening for now while this is resolved.
